projects/igniteui-angular/src/lib/grids/headers/grid-header-group.component.ts

```diff
@@ -262,8 +262,12 @@ export class IgxGridHeaderGroupComponent implements DoCheck {
      */
     @HostListener('mousedown', ['$event'])
     public onMouseDown(event: MouseEvent): void {
-        // hack for preventing text selection in IE and Edge while dragging the resize element
-        event.preventDefault();
+        if (!this.grid.allowFiltering || 
+            (event.composedPath().findIndex(el =>
+                (el as Element).tagName?.toLowerCase() === 'igx-grid-filtering-cell') < 1)) {
+                // Hack for preventing text selection in IE and Edge while dragging the resize element
+                event.preventDefault();
```

Comment:
Since this is a workaround, targeting IE and the old Edge browser engine, I wonder if it is needed. Both of those browsers are not supported anymore. Why don't you try to delete the hack and see what the behavior of the grid is then?

Reply:
I've taken the time to test the behavior of the grid without the workaround in both Chrome and Firefox. In Chrome, the first header group is being activated and all text is being selected. However, in Firefox, only the first header group is activated without any text selection.

![showcaseOfOtherBorwsers](https://github.com/IgniteUI/igniteui-angular/assets/105818882/3770f841-f6c9-4afb-9801-65c13e403014)



![showcaseOfOtherBorwsersF](https://github.com/IgniteUI/igniteui-angular/assets/105818882/c2beeb78-cbaf-4a41-9182-8458482cab04)

Reply:
😞 
I guess an issue to refactor the resizing stack to use pointer events and setPointerCapture/releasePointerCapture is in order then. This will be out of the scope of this fix which does it job.